astropy/io/fits/column.py

```diff
@@ -1047,14 +1047,16 @@ def _verify_keywords(
         # Currently we don't have any validation for name, unit, bscale, or
         # bzero so include those by default
         # TODO: Add validation for these keywords, obviously
-        for k, v in [
-            ("name", name),
-            ("unit", unit),
-            ("bscale", bscale),
-            ("bzero", bzero),
-        ]:
-            if v is not None and v != "":
-                valid[k] = v
+        valid |= {
+            k: v
+            for k, v in [
+                ("name", name),
+                ("unit", unit),
+                ("bscale", bscale),
+                ("bzero", bzero),
```

Comment:
Have you checked if the `list` fits in a single line if the trailing comma is removed?

Reply:
I did. It doesn't.